python/hsfs/transformation_function.py

```diff
@@ -17,17 +17,11 @@
 import inspect
 import ast
 
-from hsfs import util, engine
+from hsfs import util
 from hsfs.core import transformation_function_engine
 
 
 class TransformationFunction:
-    UDF = "@udf"
```

Comment:
why are these not needed anymore?

Reply:
No its not necessary any more. We use `self._spark_session.udf.register` for `selectExpr`